charts/pulsar/templates/alertmanager-statefulset.yaml

```diff
@@ -57,27 +56,69 @@ spec:
      {{- end }}
       terminationGracePeriodSeconds: {{ .Values.alert_manager.gracePeriod }}
       containers:
+      {{- if .Values.configmapReload.alertmanager.enabled }}
```

Comment:
　What is this used for? How does it work?

Reply:
this is used for reloading configuration without restarting the statefulset. when a configmap is updated, the `configmapReload` container is used for posting requests to reload configurations.